src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf16Utility.Validation.cs

```diff
@@ -433,5 +479,24 @@ static Utf16Utility()
             scalarCountAdjustment = tempScalarCountAdjustment;
             return pInputBuffer;
         }
+
+        private static readonly Vector128<byte> s_bitMask128 = BitConverter.IsLittleEndian ?
```

Comment:
@carlossanlop this throws locally when running against net472 **Debug**. We don't have Debug coverage in CI so this wasn't caught (cc @safern). This throws because Vector128.Create is stubbed out on .NETFramework.

Reply:
Oh thanks for letting me know, let me address this immediately.

Reply:
Not sure why the CI for this PR didn't catch that issue. I would've expected it to show up in either Debug or Release.

Reply:
PR to fix this: https://github.com/dotnet/runtime/pull/39652/files